build.gradle

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id "com.github.spotbugs" version "4.0.5"
+    id "com.github.johnrengelman.shadow" version "5.2.0"
+    id "de.undercouch.download" version "4.0.4"
+    id "net.researchgate.release" version "2.8.0"
+}
+
+ext.ballerinaLangVersion = project.ballerinaLangVersion
+
+allprojects {
+    group = project.group
+    version = project.version
+
+    apply plugin: 'jacoco'
+    apply plugin: 'maven-publish'
+}
+
+subprojects {
+    apply plugin: 'java'
+}
+
+def artifactCacheParent = file("${project.projectDir}/build/cache_parent/")
+def packageName = "microsoft.calendar"
+def packageOrg = "ballerinax"
+def pathToBala = file("${project.projectDir}/calendar/target/bala")
+def platform = "java11"
+
+task createBalaDirectory {
+    mkdir "${project.projectDir}/calendar/target/bala"
+}
+
+task build {
+    dependsOn('calendar:build')
+}
+
+task copyDistribution {
+    inputs.dir file(project.projectDir)
+
+    pathToBala.eachFileMatch(~/.*.bala/) { balaFile ->
+        copy {
+            from zipTree(balaFile)
+            into file("${artifactCacheParent}/bala/${packageOrg}/${packageName}/${project.version}/${platform}")
+        }
+    }
+    copy {
+        from file("${project.projectDir}/calendar/target/cache")
+        exclude '**/*-testable.jar'
+        exclude '**/tests_cache/'
+        into file("${artifactCacheParent}/cache/")
+    }
+    outputs.dir artifactCacheParent
+}
+
+task createArtifactZip(type: Zip) {
+    from "${buildDir}/cache_parent"
+    archiveName 'distribution.zip'
+    destinationDir(file("${buildDir}/distribution"))
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            artifact source: createArtifactZip, extension: 'zip'
+        }
+    }
+
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/ballerina-platform/module-ballerinax-microsoft.calendar")
```

Comment:
Seems the url of the repo is wrong.
Please correct

Reply:
Changed in all places. 
/Thanks